crates/bevy_core_pipeline/src/tonemapping/tonemapping.wgsl

```diff
@@ -0,0 +1,28 @@
+#import bevy_core_pipeline::tonemapping
+
+struct VertexOutput {
+    [[builtin(position)]]
+    position: vec4<f32>;
+    [[location(0)]]
+    uv: vec2<f32>;
+};
+
+[[stage(vertex)]]
+fn vs_main([[builtin(vertex_index)]] in_vertex_index: u32) -> VertexOutput {
+    let uv = vec2<f32>(f32((in_vertex_index << 1u) & 2u), f32(in_vertex_index & 2u));
+    let position = vec4<f32>(uv * vec2<f32>(2.0, -2.0) + vec2<f32>(-1.0, 1.0), 0.0, 1.0);
```

Comment:
So this code has uvs (0,0), (2,0), (0,2), and vertices (-1,1), (3,1), (-1,-3). I suppose it doesn't matter in this case but the winding is clockwise and we use the counter-clockwise convention for the front face. So I suppose to get UV 0,0 in the top-left, and have counter-clockwise winding, we want to have UVs (0,0), (0,2), (2,0), and positions (-1,1),(-1,-3), (3,1), which could be done as:
```suggestion
    let uv = vec2<f32>(f32(in_vertex_index >> 1u), f32(in_vertex_index & 1u)) * 2.0;
    let position = vec4<f32>(uv * vec2<f32>(2.0, -2.0) + vec2<f32>(-1.0, 1.0), 0.0, 1.0);
```